src/analysis/plugin/compat.py

```diff
@@ -58,19 +58,17 @@ def shutdown(self):
 
 def yara_match_to_dict(match: yara.Match) -> dict:
     """Converts a ``yara.Match`` to the format that :py:class:`analysis.YaraPluginBase` would return."""
-    # FIXME (yara): Use this when we upgrade to yara-python 4.3.0
-    # for string_match in match.strings:
-    #    for string_instance in string_match.instances:
-
-    strings = [(offset, identifier, data.hex()) for offset, identifier, data in match.strings]
+    # see YARA docs: https://yara.readthedocs.io/en/latest/yarapython.html#yara.StringMatchInstance
```

Comment:
Unnecessary comment?

Reply:
Hmm I think it is useful since you can't look at the YARA source (because it is a compiled C library)